playground/attention_alibi_template.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2025 The IREE Authors
```

Comment:
Can we move this out of playground? I was thinking it may be good to have it in 
`iree/turbine/kernel/wave/template/attention/flexattention/`  wdyt @harsh-nod ?

Reply:
Moved to a file next to other attentions. FlexAttention is slightly broader and will need some config on top to select a combination of the attention variations (alibi, rpe, causal, softcap, etc.) that may require some changes in core to, e.g., gracefully handle lack of uses for certain arguments.

Reply:
Sure that's fine for now. When RPE comes in, we can refactor this into a flexattention folder and generalize this.